salt/modules/yaml.py

```diff
@@ -0,0 +1,50 @@
+"""
+Yaml helper module for troubleshooting yaml
+
```

Comment:
can you document that the usage of this module requires yamllint is installed.

and add a versionadded for 3005